src/mobile/src/components/Post/Posts.jsx

```diff
@@ -0,0 +1,53 @@
+import Constants from 'expo-constants';
+import { FlatList, View, Button, StyleSheet } from 'react-native';
+import useSWRInfinite from 'swr/infinite';
+import Post from './Post';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  divider: {
+    backgroundColor: '#9f9f9f',
+    height: 1,
+    marginHorizontal: 'auto',
+    marginVertical: '1rem',
+    width: '80%',
+  },
+});
+
+const POST_URL = 'https://dev.api.telescope.cdot.systems/v1/posts'; //Constants.manifest.extra.postsURL;
```

Comment:
Why are we not using `Constants.manifest.extra.postsURL`? I see the comment

Reply:
I try to implement it but somehow it is not working for me

Reply:
I see